pandas/tests/series/indexing/test_boolean.py

```diff
@@ -70,14 +70,6 @@ def test_getitem_boolean_object(string_series):
     s2[mask] = 5
     tm.assert_series_equal(cop, s2)
 
-    # nans raise exception
```

Comment:
This one was checking that an _object_ indexing array containing `bool` and `NaN` values raises, which I think should still be the case even if a nullable boolean does not?

Reply:
That sounds right.